### ISSUE TITLE
Outdated Name suggestions

### ISSUE THREAD

**Issue by SK53** [openstreetmap/iD#6523]
I've just being doing some shop editing from slightly old survey data and I found a couple of brands in the name suggestion index which are very unlikely (shops have closed or been re-branded). I'll raise the specifics over at the name-suggestion-index, but what interests me is the ability to highlight shop names which are unlikely for these two reasons. Robert Whittaker maintains a page of 'Ghosts' in the UK at https://osm.mathmos.net/ghosts/. I am now wondering if such things could also be raised for editors in iD. Presumably it would require some kind of addition to name suggestions to mark those which are now outdated (& why).

It may be regarded as inappropriate for iD because users are unlikely to have the objects in front of them, and therefore may encourage 'fixing' things unseen. So this is very much one to ponder on.

**Comment by bhousel** [comment]
related:  https://github.com/osmlab/name-suggestion-index/issues/2415

**Comment by quincylvania** [comment]
Since https://github.com/osmlab/name-suggestion-index/pull/2926 was merged, iD can now consume [dist/dissolved.json](https://github.com/osmlab/name-suggestion-index/blob/master/dist/dissolved.json) and issue warnings in specific countries after the specified dates.

**Comment by matkoniecz** [comment]
tl;dr it seems to be at https://cdn.jsdelivr.net/npm/name-suggestion-index@latest/dist/wikidata/dissolved.json - despite some NSI documentation claiming otherwise

when testing should be likely mixed with https://cdn.jsdelivr.net/npm/name-suggestion-index@latest/dist/json/nsi.json

is it fine from  legal and licensing perspective to make validator warnings based on Wikidata (yes, they claim CC0, but has they decided on database rights one way or another?)?

> can now consume [dist/dissolved.json](https://github.com/osmlab/name-suggestion-index/blob/master/dist/dissolved.json) and issue warnings in specific countries after the specified dates.

has this got abandoned? I see last update in 2020.

https://github.com/osmlab/name-suggestion-index/wiki/Distributed-Files claims it is up to date

https://github.com/osmlab/name-suggestion-index/wiki/Downloading-the-Data claims that other documentation page is wrong

https://cdn.jsdelivr.net/npm/name-suggestion-index@latest/dist/wikidata/dissolved.json has more entries
